src/it/projects/MSHADE-420/verify.groovy

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import java.security.MessageDigest
+import java.util.jar.JarFile
+
+String describeEntry(JarFile jar, String name, long time) {
+    def entry = jar.getEntry(name)
+    return String.format("  - %-47s: time = ", name) + entry.getTime() + ", lastModified = " + entry.getLastModifiedTime()\
```

Comment:
Why don't you use `String#format()` throughout?

Reply:
tested with more recent JDK and the output was ok :)
going to a simple boolean, which is what really this display is about (not really value details)

Reply:
good question: done